controls/package_spec.rb

```diff
@@ -80,7 +80,7 @@
   impact 1.0
   title 'Install auditd'
   desc 'auditd provides extended logging capacities on recent distribution'
-  audit_pkg = os.redhat? || os.suse? ? 'audit' : 'auditd'
+  audit_pkg = os.redhat? || os.suse? || os.family == 'amazon' ? 'audit' : 'auditd'
```

Comment:
InSpec is going to use the helper like `linux?` or `windows?` for family. Lets use `os.name == 'amazon'` here